test/development/acceptance/ReactRefreshLogBox-builtins.test.ts

```diff
@@ -49,9 +49,6 @@ describe.each(['default', 'turbo'])('ReactRefreshLogBox %s', () => {
       expect(await session.getRedboxSource()).toMatchInlineSnapshot(`
         "./node_modules/my-package/index.js:1:13
         Module not found: Can't resolve 'dns'
-        > 1 | const dns = require('dns')
```

Comment:
For node_modules the codeframe should still be shown, just not for ones that are very few lines and the individual line is thousands of characters.

Reply:
This is consistent with webpack's behavior — should we change both?